manifests/charts/base/README.md

```diff
@@ -1,6 +1,6 @@
 # Istio base Helm Chart
 
-This chart installs resources shared by all Istio revisions. This includes Istio CRDs.
+This chart installs core components required for any configuration of Istio to function - primarily and notably the Istio CRDs.
```

Comment:
*really pedantic nit*:
"any configuration of Istio" isn't exactly true. You can already get a functional install of ambient without adding the Istio CRDs if you are really looking to be fully gtwapi. You could likely just install peerAuthn and authzPol and get yourself really far for basic use cases. Istiod actually doesn't really care about istio crds existing anymore since folks have been using delayed informers for (at least near as makes no difference for ambient) everything.

Reply:
then `base` is just the wrong name for this chart, no matter how you slice it.

Reply:
The original description description was really accurate (at least prior to this PR) -- base was the things that there is exactly 1 of

Reply:
That's fair - the name `base` implies it's a required subset of Istio functionality (mostly not true).

The description is partially true, but less so with ambient (these are things required for _sidecar_ revisions, many of them are not required outside of that context)

Reply:
Actually that's not even true - multiple resources in here are created per-revision actually.

So this is sort of a mismash chart.